### ISSUE TITLE
FM代码的一处细节疑问

### ISSUE THREAD

**Issue by TangDouOVO** [DSXiangLi/CTR#7]
你好！FM代码中“interaction_term = 0.5 * tf.reduce_mean(sum_square - square_sum, axis=1, keep_dims= True)”
论文中用的是求和，这里用的是求均值。是考虑到k不同，且k选取越大、二阶交叉项越大，所以才取的均值吗？为了消除不同k取值的影响？

**Comment by DSXiangLi** [comment]
哈哈不是，这里确实应该是sum的我去改一下。影响应该不大这里除的常数，旁边的linear_term也能学到。

**Comment by TangDouOVO** [comment]
谢谢！
